src/components/Info.js

```diff
@@ -0,0 +1,77 @@
+import React from 'react';
+import styled from 'styled-components';
+
+const Article = styled.article`
+  width: 100%;
+  max-width: 739px;
+  margin: 130px auto 0;
+  padding: 0 20px 185px;
+
+  > p {
+    line-height: 26.98px;
+    letter-spacing: 0.03px;
+  }
+`;
+
+const Heading = styled.h2`
+  font-size: 24px;
+  font-family: ${({ theme }) => theme.font.secondary};
+  color: ${({ theme }) => theme.color.secondary};
+  margin-bottom: 14px;
+`;
+
+const List = styled.ul`
+  margin: 0px 0 105px;
+  padding: 0;
+  list-style-position: inside;
+`;
+
+const ListItem = styled.li`
+  font-size: 10px;
+  margin-left: 1px;
+  line-height: 25px;
+  > span {
+    position: relative;
+    left: -5px;
+    font-size: ${({ theme }) => theme.font.size.normal};
+  }
+`;
+
+const ExternalLink = styled.a`
+  text-decoration: none;
+  color: ${({ theme }) => theme.color.externalLink};
+`;
+
+const Info = () => (
+  <Article>
+    <Heading id="how-it-works">How it works</Heading>
+    <List>
+      <ListItem>
+        <span>We find the 500 top posts from the past year for a subreddit.</span>
+      </ListItem>
+      <ListItem>
+        <span>
+          The data is visualized in a heatmap grouped by weekday and hour of the
+          day.
+        </span>
+      </ListItem>
+      <ListItem><span>See immediately when to submit your reddit post.</span></ListItem>
+    </List>
```

Comment:
You could extract this part into a separate component called `InfoSection` or so that is rendered like the following and re-use it below. The benefit is not so great here, but it's still a nice abstraction imo. If anyone needs to add another info section at some point they could just use that component.

```
<InfoSection headline="How it works">
  <List>
    ...
  </List>
</InfoSection>
```

Reply:
Makes sense!

Question:  When I pass props, I get eslint warnings:

`[prop] is missing in props validationeslintreact/prop-types`

If I add propsTypes, I then get another message saying they are missing a default value.

Should I ignore these altogether?  Or go ahead and do props validation?